metals/src/main/scala/scala/meta/internal/metals/doctor/Doctor.scala

```diff
@@ -251,12 +257,26 @@ final class Doctor(
       .render
   }
 
+  /**
+   * Java 8 is deprecated and in the future Metals will require Java >=11
+   */
+  private def deprecatedJavaVersion(): Option[String] = for {
+    versionString <- Option(System.getProperty("java.version"))
+    version <- JdkVersion.parse(versionString)
+    if version.major < 11
+  } yield s"Running Metals on Java ${version.major} is deprecated. In the future Metals will require Java 11"
```

Comment:
Can we say "... will require Java 11 or higher"? So that users won't misunderstand Metals requires exactly Java 11.

Reply:
Good call!